.github/workflows/test.yml

```diff
@@ -26,3 +26,9 @@ jobs:
       run: |
         chmod +x ci/test.sh
```

Comment:
Useless `chmod` command.

Reply:
fixed in 6cec3ca1417d268a544349d1f4a3c9ebfb386912